src/functional/program_processors/runners/gtfn_cpu.py

```diff
@@ -41,6 +41,7 @@ class GTFNExecutor(ppi.ProgramExecutor):
     builder_factory: compiler.BuildSystemProjectGenerator = compiledb.CompiledbFactory()
 
     name: Optional[str] = None
+    apply_transforms: bool = True  # TODO replace by more general mechanism, see https://github.com/GridTools/gt4py/issues/1135
```

Comment:
I'd consider renaming this variable. `apply_transforms` is obvious if you've already seen the implementation, but if you come from the top it might not be clear what transforms are applied to what. Transforming the arguments to column-major memory layout would also be possible.
```suggestion
    apply_itir_transforms
    apply_itir_passes
    enable_itir_passes
    enable_itir_optimizations
    enable_optimizations
```

Reply:
I chose `enable_itir_transforms`